newsroom/topics/topics.py

```diff
@@ -19,7 +19,8 @@ class TopicsResource(newsroom.Resource):
         'is_global': {'type': 'boolean', 'default': False},
         'subscribers': {
             'type': 'list',
-            'schema': newsroom.Resource.rel('users', required=True)
+            'schema': newsroom.Resource.rel('users'),
+            'required': True
```

Comment:
why are subscribers required?

Reply:
There was an issue while updating the topics. Where subscribers was required. Fixed it and updated the PR.